packages/repository-tests/src/crud/relations/acceptance/has-many-through.relation.acceptance.ts

```diff
@@ -0,0 +1,99 @@
+// Copyright IBM Corp. 2020. All Rights Reserved.
+// Node module: @loopback/repository-tests
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {expect, toJSON} from '@loopback/testlab';
+import {
+  CrudFeatures,
+  CrudRepositoryCtor,
+  CrudTestContext,
+  DataSourceOptions,
+} from '../../..';
+import {
+  deleteAllModelsInDefaultDataSource,
+  MixedIdType,
+  withCrudCtx,
+} from '../../../helpers.repository-tests';
+import {
+  Customer,
+  CustomerRepository,
+  Order,
+  OrderRepository,
+  Seller,
+  SellerRepository,
+} from '../fixtures/models';
+import {givenBoundCrudRepositories} from '../helpers';
+
+export function hasManyRelationAcceptance(
+  dataSourceOptions: DataSourceOptions,
+  repositoryClass: CrudRepositoryCtor,
+  features: CrudFeatures,
+) {
+  describe('HasManyThrough relation (acceptance)', () => {
+    before(deleteAllModelsInDefaultDataSource);
+    let customerRepo: CustomerRepository;
+    let orderRepo: OrderRepository;
+    let sellerRepo: SellerRepository;
+    let existingCustomerId: MixedIdType;
+
+    before(
+      withCrudCtx(async function setupRepository(ctx: CrudTestContext) {
+        ({customerRepo, orderRepo, sellerRepo} = givenBoundCrudRepositories(
```

Comment:
Why enclose within `()`?

Reply:
Short notation of
```ts
const {customerRepo, orderRepo, sellerRepo} = givenBoundCrudRepositories(
  ctx.dataSource,
  repositoryClass,
  features,
);
```
same as here:
https://github.com/strongloop/loopback-next/blob/098dc971f1fcb00321ca9226a2475c62aa5d00d0/packages/repository-tests/src/crud/relations/acceptance/has-many.relation.acceptance.ts#L42